angular/cmbs-property-summary/src/app/property-locations/property-locations.component.ts

```diff
@@ -16,22 +16,19 @@ import { DataSource } from "@angular/cdk/collections";
 })
 export class PropertyLocationsComponent implements OnInit {
   stateSummary: UsSummary[];
-  assetDetail: UsSummary[];
 
   title: string = 'My first AGM project';
```

Comment:
I copied this from the AGM getting started guide.  Does the title get used anywhere? If so we should give it a meaningful name otherwise lets drop it.